.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+on: [push]
+
+jobs:
+  lint:
+    name: Lint on ${{ matrix.node-version }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [10]
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm install, and pretest
+      run: |
+        npm install
+        npm run pretest
+      env:
+        CI: pretest
+
+  test:
+    name: Test on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [8, 10, 12]
```

Comment:
could you check will our test work with `*-alpine` node, it extremely reduce image size
<img width="931" alt="Снимок экрана 2019-09-17 в 12 22 39" src="https://user-images.githubusercontent.com/10380560/65029169-ed606880-d945-11e9-95f0-e8fdaf92a6b9.png">

You can check all available images [here](https://hub.docker.com/_/node)

Reply:
Currently this affect only `node-8` since `node-8` and `node-12` are cached on github actions servers, see https://help.github.com/en/articles/software-in-virtual-environments-for-github-actions

Reply:
Oh it does not work this way =) `setup-node` install node using `node-12` 😎